### PR TITLE
Lk wdl best practices

### PR DESCRIPTION
**Pull request by ekiernan**
This PR adds a WDL Best Practices Section to the WARP doc site and the first Best Practices doc - optimizing costs for GC.

### CODE REVIEW

**Review by kayleemathews — APPROVED** [review]
This LGTM to me! Thanks Liz!

**Review by kevinpalis — APPROVED** [review]
Looks good!

**Review by jessicaway — APPROVED** [review]
A couple minor comments, but looks great! Thank you @ekiernan!

**Review by khajoue2 — APPROVED** [review]

File: website/docs/Best_practices/GC_cost_optimization.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+--runThreadN ~{cpu}
+--genomeDir genome_reference
+>>>
+```
+
+In the use case above, it makes sense to combine the bash operations and STAR. The bash operations are only used to make a new directory for the genome references and to compress files. After the files are ready, the heavy lifting of genome alignment is done by STAR.
+
+Keep in mind there might be scenarios where splitting out more intensive bash operations might make sense. For example, if your bash operations are time intensive and your tool is not very reliable, you might want to save the outputs of your bash functions before moving on to running the tool. In this case, it would make sense to keep the bash commands and your tool in separate WDL tasks to avoid rerunning them both.
+
+## Tip 3: Avoid input/output timing costs (moving and loading lots of files)
```
[the remark this answers is not in the document]
Comment (ekiernan) [reply website/docs/Best_practices/GC_cost_optimization.md:48]: Thanks, Jess! I added a section for this point

File: website/docs/Best_practices/GC_cost_optimization.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+Just like starting a VM has some overhead cost, localizing files also has some downsides, including using more time in your VM. And the more time you spend, the more your cost will go up. Your VM needs to find all your cloud files in order to run them through your different tools. That means if you have thousands of files, you’ll have to run initialization steps for each of them, which can take a bit of time and networking. 
+
+Each time you move cloud files, you pay egress for the network transitions, so it’s important to find the balance in the number of files you decide to move. You have to weigh whether it costs more to move a large file vs. moving several smaller files. For example, you might find that it’s more cost-efficient to move a zipped ~100 GB file than to move 100, 1 GB files. After running a  test workflow, check your workflow logs to see what the timing is for localizing and moving files vs. running your tool. This might require trial and error when developing your workflow. 
+
+## Tip 4: Run files in parallel when possible
```
[the remark this answers is not in the document]
Comment (ekiernan) [reply website/docs/Best_practices/GC_cost_optimization.md:54]: I added a paragraph for this point

File: website/docs/Best_practices/GC_cost_optimization.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jessicaway) [reply website/docs/Best_practices/GC_cost_optimization.md:50]: Looks great, thanks @ekiernan!

File: website/docs/Best_practices/GC_cost_optimization.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jessicaway) [reply website/docs/Best_practices/GC_cost_optimization.md:56]: 👍 thank you!

**Comment by nikellepetrillo** [comment]
retest this please

**Comment by jessicaway** [comment]
retest this please

**Pull request merged**